external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -309,4 +309,25 @@
 
   </Target>
 
+  <Target Name="AddRemoteExecutorHostTools"
```

Comment:
I understand we would want this change but I'm not sure how is this related to ApiCompat changes. I assume this is just needed because of other Arcade changes?

Reply:
I deleted it.  It was reacting to other arcade changes.  See @ViktorHofer and my comments above.